release_notes.md

```diff
@@ -1,3 +1,48 @@
+*******************************
+# Release Notes for Dymos 1.7.0
+
+February 09, 2023
+
+Version 1.7.0 contains some significant enhancements to its capabilities.
+
+Major improvements include:
+- Users may now specify boundary constraints, path constraints, and the objective within a phase as expressions. These expressions must contain an equals sign and must be complex-step-safe. This allows the user to impose constraints or objectives on quantities that may not be computed by the given equations of motion model.
+- While the user could always time as some other quantity being used as the variable of integration (such as range for an aircraft), the user can now explicitly rename the time variable to make this a little more obvious.
```

Comment:
Reword this one. Maybe;
"The user can now rename the time variable. Integration on another variable (such as range for an aircraft) has always been possible, but dymos still always called the variable "time".  Now, it will be more clear.